src/main/resources/tinylog.properties

```diff
@@ -11,3 +11,12 @@ exception = strip: jdk.internal
 level@org.jabref.gui.maintable.PersistenceVisualStateTable = debug
 
 level@org.jabref.http.server.Server = debug
+
+level@org.jabref.logic.ai.AiIngestor = trace
+
+# TODO: Set up properly.
```

Comment:
The statements won't work. They are processed by tinylog, but not by langchain. You need to follow the setup-by-code of langchain. Just follow "PureJava" from https://docs.langchain4j.dev/tutorials/logging/#pure-java.